staging/src/k8s.io/apiserver/pkg/storage/etcd3/watcher_test.go

```diff
@@ -32,6 +32,8 @@ import (
 func TestWatch(t *testing.T) {
 	ctx, store, _ := testSetup(t)
 	storagetesting.RunTestWatch(ctx, t, store)
+	storagetesting.TestClusterScopedWatch(ctx, t, store)
```

Comment:
nit - while this technically works, I would prefer to have explicit separate tests for this, i.e.:

```
func TestClusterScopedWatch(t *testing.T) {
   ...
}

func TestNamespaceScopedWatch(...) {
  ....
}
```

This makes it easier to immediately say what exactly failed.

Reply:
Done.